web/src/components/Upstream/components/UpstreamType.tsx

```diff
@@ -0,0 +1,62 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import React from 'react';
+import { Form, Select } from 'antd';
+import { useIntl } from 'umi';
+import type { FormInstance } from 'antd/lib/form';
```

Comment:
```suggestion
import type { FormInstance } from 'antd/lib/form';

```

Reply:
@LiteSun In fact, I found its various in old codes uses through global code search. such as `antd/lib/form`, `antd/es/form` and `antd`. Which one should I choose? Thx